datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/dataplatforminstance/mappers/DataPlatformInstanceMapper.java

```diff
@@ -33,6 +47,39 @@ public DataPlatformInstance apply(@Nonnull final EntityResponse entityResponse)
     MappingHelper<DataPlatformInstance> mappingHelper = new MappingHelper<>(aspects, result);
     mappingHelper.mapToResult(DATA_PLATFORM_INSTANCE_KEY_ASPECT_NAME, this::mapDataPlatformInstanceKey);
 
+    final EnvelopedAspect envelopedDataPlatformInstanceProperties = aspects.get(Constants.DATA_PLATFORM_INSTANCE_PROPERTIES_ASPECT_NAME);
+    if (envelopedDataPlatformInstanceProperties != null) {
+      result.setProperties(mapDataPlatformInstanceProperties(
+              new DataPlatformInstanceProperties(envelopedDataPlatformInstanceProperties.getValue().data()), entityUrn)
+      );
+    }
+
+    final EnvelopedAspect envelopedOwnership = aspects.get(Constants.OWNERSHIP_ASPECT_NAME);
+    if (envelopedOwnership != null) {
+      result.setOwnership(OwnershipMapper.map(new Ownership(envelopedOwnership.getValue().data()), entityUrn));
+    }
+
+    final EnvelopedAspect envelopedTags = aspects.get(Constants.GLOBAL_TAGS_ASPECT_NAME);
+    if (envelopedTags != null) {
+      com.linkedin.datahub.graphql.generated.GlobalTags globalTags = GlobalTagsMapper.map(new GlobalTags(envelopedTags.getValue().data()), entityUrn);
+      result.setTags(globalTags);
```

Comment:
while `globalTags` is deprecated, I'm not positive we've migrated everywhere off of the frontend from using it. could you do `result.setGlobalTags` here as well as `result.setTags`?

Reply:
There is no `result.setGlobalTags` in the `com.linkedin.datahub.graphql.generated.DataPlatformInstance`. 

In https://github.com/datahub-project/datahub/pull/5728 I added the `GlobalTags` aspect in the `entity-registry.yml`, and skipped the deprecated one for `Tags`. With that, the graphql generator produces the following methods about tags:

```
    /**
     * Tags used for searching the data platform instance
     */
    public GlobalTags getTags() {
        return tags;
    }
    /**
     * Tags used for searching the data platform instance
     */
    public void setTags(GlobalTags tags) {
        this.tags = tags;
    }
```

So there is no `setGlobalTags` and instead the `setTags` is about `GlobalTags`.

Do you still see any issue on this?

Reply:
oh nice! okay that all makes sense and is good to go then.